.changeset/lucky-dogs-follow.md

```diff
@@ -0,0 +1,5 @@
+---
+"kilo-code": patch
+---
+
+The OpenRouter provider now uses the custom base URL when fetching the model list.
```

Comment:
Do we know the real use case?  Because this seems quite convenient for _us_, since it might well render quite a bit of other kilo-provider stuff obsolete, but since it's coming from an outside contributor, I'm really curious.

Reply:
I will follow up with the PR creator